.changeset/bright-icons-flash.md

```diff
@@ -0,0 +1,44 @@
+---
+"@fluidframework/container-runtime": minor
+"@fluidframework/core-interfaces": minor
+"@fluidframework/datastore-definitions": minor
+"fluid-framework": minor
+"@fluidframework/fluid-static": minor
+"@fluidframework/runtime-definitions": minor
+"@fluidframework/shared-object-base": minor
+"@fluidframework/tree": minor
```

Comment:
Feels like it just be major in most places. Not sure it matters (based on where are with our releases).

Reply:
Our current workflow doesn't use this in a user facing way, and leaving it at the default (computed by the changeset tool) seems to be the simplest option for our current release process. Its not ideal, but it will work more logically once we are post 2.0 (and then the main branch defaulting to minor will be correct data we don't use instead of ambiguous data we don't use).